test/e2e/specs/specs-playwright/wp-signup__domain.ts

```diff
@@ -0,0 +1,134 @@
+/**
+ * @group calypso-release
+ */
+
+import {
+	DataHelper,
+	DomainSearchComponent,
+	setupHooks,
+	BrowserManager,
+	CloseAccountFlow,
+	UserSignupPage,
+	CartCheckoutPage,
+	NavbarComponent,
+	IndividualPurchasePage,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe( DataHelper.createSuiteTitle( 'Signup: WordPress.com Domain Only' ), function () {
+	const inboxId = DataHelper.config.get( 'signupInboxId' ) as string;
+	const username = `e2eflowtestingdomainonly${ DataHelper.getTimestamp() }`;
+	const email = DataHelper.getTestEmailAddress( {
+		inboxId: inboxId,
+		prefix: username,
+	} );
+	const signupPassword = DataHelper.config.get( 'passwordForNewTestSignUps' ) as string;
+
+	let page: Page;
+	let domainSearchComponent: DomainSearchComponent;
+	let cartCheckoutPage: CartCheckoutPage;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	describe( 'Signup via /start/domain', function () {
+		it( 'Navigate to /start/domain', async function () {
+			await page.goto( DataHelper.getCalypsoURL( '/start/domain' ) );
+		} );
+
+		it( 'Set store cookie', async function () {
+			await BrowserManager.setStoreCookie( page, { currency: 'JPY' } );
+		} );
+
+		it( 'Search for a domain', async function () {
+			domainSearchComponent = new DomainSearchComponent( page );
+			await domainSearchComponent.search( username + '.live' );
+		} );
+
+		it( 'Select a .live domain', async function () {
+			await domainSearchComponent.selectDomain( '.live' );
+		} );
+
+		it( 'Select to buy just the domain', async function () {
+			await page.click( 'button:text("Just buy a domain")' );
+		} );
+
+		it( 'Sign up for a WordPress.com account', async function () {
+			const userSignupPage = new UserSignupPage( page );
+			await userSignupPage.signup( email, username, signupPassword );
+		} );
+
+		it( 'Land in checkout cart', async function () {
+			cartCheckoutPage = new CartCheckoutPage( page );
+			const totalAmount = await cartCheckoutPage.getCheckoutTotalAmount();
+			expect( totalAmount ).toBeGreaterThan( 0 );
+		} );
+
+		it( 'Prices are shown in Japanese Yen', async function () {
+			const buttonText = await cartCheckoutPage.getPaymentButtonText();
+			buttonText.startsWith( '¥' );
```

Comment:
Looks like this should be wrapped with `expect`

Reply:
Haha, oops...

Reply:
Found out this snippet was not actually doing anything useful, and it would have failed if wrapped with `expect` because the actual button text reads something like `Pay A$120`, so the assertion will always fail.

I've had to rewrite this step to read the value from the checkout total, but the resulting step runs and passes as expected.